deny.toml

```diff
@@ -63,6 +63,14 @@ deny = [
     # Having multiple versions of this can silently cause images (such as
     # the logo in the about dialog) in egui to not appear.
     { name = "image", deny-multiple-versions = true },
+    # Force using the 1.0.x series, with no actual IDNA support. We don't
+    # need it (Flash Player didn't support it), so it would just add bloat.
```

Comment:
This statement is yet to be verified still.

Reply:
IDNA domain names seem to work on (my) Linux, but not on Windows (10)... So I think it's fine it they don't work in Ruffle.

Reply:
What would happen if a IDNA domain embedds rust? Does that break?

Reply:
I don't know how to test it, but my understanding was that we'd propagate the `http://xn--stuff` string all the way to the `fetch` api, which would handle it correctly?
Do I get this right?

Reply:
I also have no idea, nor about how to check. :/ I guess I could either buy an `xn--` domain, try to find one with embedded flash, or inject an embedding into a site on one...?
But yeah, my guess is the same: since FP on Windows desktop doesn't understand IDNA (as shown below), probably the browser plugin didn't either... And whatever transformations the browser did for it (if any), we could probably utilize as well.

Reply:
Oh... Guess what `/etc/hosts` is for... :man_facepalming: :sweat_smile:

Reply:
I test this using /etc/hosts and it seems to work. :tada: